### ISSUE TITLE
[Bug]: Chromatic and MultiSelect

### ISSUE THREAD

**Issue by CodyCBakerPhD** [NeurodataWithoutBorders/nwb-guide#51]
As seen from last trigger of Chromatic, complains about the MultiSelect form

```
Could not resolve "./multiselect/index.js" from "src/stories/Multiselect.stories.js"
```

https://github.com/NeurodataWithoutBorders/nwb-guide/actions/runs/4481833739/jobs/7879085476

**Comment by CodyCBakerPhD** [comment]
Was run on the branch for #50 if that helps any

**Comment by garrettmflynn** [comment]
@CodyCBakerPhD This should be addressed in #53.
